### ISSUE TITLE
Notepad++ useless when network drive unavailable

### ISSUE THREAD

**Issue by jhdoran** [notepad-plus-plus/notepad-plus-plus#4877]
### Description of the Issue
When a network drive became unavailable, Notepad++ begin asking if tabs should be removed.  This is ALL it did.  After about 5 minutes of 100% unavailability, I killed the program using Task Manager.  
Drives go away.  Don't force me to answer prompts and wait while Notepad++ is unresponsive for minutes, until I've forgotten what I was going to type.

### Steps to Reproduce the Issue

1.   Open some tabs on a network drive
2.   Disconnect the network drive
3.   Kill Notepad++ with taskmanager

### Expected Behavior

The dialog for removing a tab should allow me to ignore the rest of the messages.  I can reconnect the drive later.  Label this button STFU.

### Actual Behavior

A dialog appears.  I close it.  Notepad is unresponsive for 15 seconds or so.  Another dialog appears.  Rinse and repeat.

### Debug Information
None collected.  I terminated the process.

**Comment by mjmatthiesen** [comment]
I'd like to add that what I suspect is happening is a network communication timeout. npp is waiting for 5+ minutes on a response before displaying the following dialogue:
![image](https://user-images.githubusercontent.com/16603906/55574077-ba445880-56c8-11e9-8ca0-ea45c75a6dce.png)

Honestly, waiting more than 10-15 seconds is entirely useless. If it isn't available in that time frame, move on, no need to kill the whole app.

Or at the very least, launch that network query in a thread that doesn't freeze the entire app????

**Comment by MetaChuh** [comment]
@z3rone 

a freeze issue and repetitive "keep file" alerts are fixed on the most recent version 7.6.6.

if multiple files are loaded from a mapped network drive and/or unc path, and the network cable or wifi connection is disabled or the connection has deteriorated completely, notepad++ will alert you "the file does not exist anymore. keep this file in editor?", on the current tab, but not for all others, except when switching tabs to that specific document.

please update your notepad++ to the latest version [<kbd>Notepad++ 7.6.6</kbd>](https://notepad-plus-plus.org/download/v7.6.6.html) and migrate or reinstall your plugins if you update from version 7.5.9 or below.

thank you for your help and best regards.

-----
```
general notes:

please close your issues as soon as you are satisfied, to help
keeping the notepad++ issue tracker clean.

when creating a new issue, please provide all information as shown at the issue template.
your notepad++ debug info is part of the minimum requirements.
(? > Debug Info... > Copy debug info into clipboard).

please also conduct a search at the issue tracker beforehand, to check if a similar issue
already exists.

for general questions, or if it is not sure, whether your issue is directly related to
the notepad++ source code, please visit us at the notepad++ community forum, and
search if related topics exists. you are welcome to post either at similar
topics, or to create a new topic at Help Wanted or General Discussion.
(no extra account is needed, just use your github account to sign in)
```
[<p align="center"><kbd> &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; Click here to visit the Notepad++ Community Forum &emsp; &emsp; &emsp; &emsp; &emsp; &emsp;&emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; </kbd></p>](https://notepad-plus-plus.org/community/)
